apps/api/prisma/schema.prisma

```diff
@@ -8,52 +8,55 @@ datasource db {
 }
 
 model Item {
-  id             Int        @id @default(autoincrement())
-  parent_item    ParentItem @relation(fields: [parent_item_id], references: [id])
-  parent_item_id Int
-  name           String
-  sku            String
-  price          Decimal
-  quantity       Int
-  created_at     DateTime?
-  updated_at     DateTime?
+  id           Int        @id @default(autoincrement())
+  parentItem   ParentItem @relation(fields: [parentItemId], references: [id])
+  parentItemId Int        @map("parent_item_id")
+  name         String
+  sku          String
+  price        Decimal
+  quantity     Int
+  created_at   DateTime?
+  updated_at   DateTime?
+
+  purchaseOrderLineItems PurchaseOrderLineItem[]
 
   @@map("items")
 }
 
 model ParentItem {
-  id         Int       @id @default(autoincrement())
-  name       String
-  created_at DateTime?
-  updated_at DateTime?
+  id        Int       @id @default(autoincrement())
+  name      String
+  createdAt DateTime? @map("created_at")
+  updatedAt DateTime? @map("updated_at")
 
   items Item[]
 
   @@map("parent_items")
 }
 
-model PurchaseOrderLineItems {
-  id                Int            @id @default(autoincrement())
-  purchase_order    PurchaseOrders @relation(fields: [purchase_order_id], references: [id])
-  purchase_order_id Int
-  item_id           Int
-  quantity          Int
-  unit_cost         Decimal
-  created_at        DateTime?
-  updated_at        DateTime?
+model PurchaseOrderLineItem {
+  id              Int           @id @default(autoincrement())
+  purchaseOrder   PurchaseOrder @relation(fields: [purchaseOrderId], references: [id])
+  purchaseOrderId Int           @map("purchase_order_id")
+  item            Item          @relation(fields: [itemId], references: [id])
```

Comment:
Added a relation to `Item` so we could grab those for the full purchase order detail page.